strapi/strapi
# Fix error handling for transfer websocket connection

### What does it do?

- captures connection error, when opening a websocket and throws a clearer message, (wrong token or connection error)
- It mainly fixes the issue when a user provides a wrong url or a wrong token and makes the errors clearer
- waits for a connection to be open before proceeding with transfer steps

### Why is it needed?

- make errors more clear

### How to test it?

- send a wrong url in the transfer should throw this error
<img width="630" alt="image" src="https://user-images.githubusercontent.com/37274596/222436927-fedf023b-94fe-4db8-b1dd-b5a51926723b.png">
-send a wrong token in the transfer should throw this error
<img width="630" alt="image" src="https://user-images.githubusercontent.com/37274596/222437117-0e64ce84-5812-4404-a026-a8ebc9ffced7.png">
- if you send everything correctly, the transfer should work correctly
